.gitmodules

```diff
@@ -25,3 +25,6 @@
 [submodule "apps/relap-7"]
 	path = apps/relap-7
 	url = git@github.inl.gov:ncrc/relap-7.git
+[submodule "apps/moose"]
+	path = apps/moose
+	url = git@github.com:idaholab/moose.git
```

Comment:
this is the correct address right?
is there some sort of firewall rule on civet or something preventing me from adding a submodule ?
@loganharbour